### ISSUE TITLE
Multi-Attach error

### ISSUE THREAD

**Issue by tpoxa** [onyx-dot-app/onyx#996]
I am trying to install danswer into kubernetes cluster.
Problem that two different pods (api-server and background server) appear to be using the same volumes.
Can this be omitted?
Do they both have to write there or some service only read?

thanks

**Comment by voarsh2** [comment]
Can you make use of RXW volumes?

**Comment by tpoxa** [comment]
@voarsh2 yes, we can use `ReadWriteMany` easiest way of doing it in GCP was using of Filestore (managed NFS server)
Each PVC creates 1TB instance (minimal size allowed) of Filestore which is quite expensive. So why do we need two volumes? Can it be the same volume so danswer will just use different filesystem paths? Is it supported? Thanks.

**Comment by tpoxa** [comment]
I see they are already using different mount paths

**Comment by al-lac** [comment]
@tpoxa did you manage to solve this without switching to `ReadWriteMany`?

**Comment by al-lac** [comment]
Ah ok. For me it works if i scale everything up at the same time. But if one of the pods of api-server or background-server goes down, it cannot come back up.
